pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1846,6 +1850,13 @@ func buildListener(opts buildListenerOpts) *xdsapi.Listener {
 		FilterChains:    filterChains,
 		DeprecatedV1:    deprecatedV1,
 	}
+
+	if util.IsIstioVersionGE13(node) && (opts.needHTTPInspector || needTLSInspector) {
```

Comment:
dont nneed the inspector checks. The listener filter timeout will be ignored if there are no listener filters.